tests/bofire/data_models/objectives/test_desirabilities.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Tuple
```

Comment:
Move the content of this file to https://github.com/experimental-design/bofire/blob/main/tests/bofire/data_models/specs/objectives.py. When you add it there, it is automatically tested for serialization, deserialization and validation errors.